cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,21 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+PROJECT?="google_containers"
```

Comment:
nit: I'd suggest keeping REGISTRY above PROJECT, just to stick with the hierarchy.